DittoToolsApp/DittoToolsApp.xcodeproj/project.pbxproj

```diff
@@ -784,7 +784,7 @@
 			repositoryURL = "https://github.com/getditto/DittoSwiftPackage";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 4.5.3;
+				minimumVersion = 4.5.0;
```

Comment:
I aligned this to the version specified in the `Package.swift` of the `DittoSwiftTools` package, but longer term I wonder if we want to explicitly include this dependency here or let it be pulled transitively by the fact that we include the local `DittoSwiftTools` package which has the dependency already.

Pros of including it this way:
- Allows for compatibility testing with specific versions

Cons
- An extra spot to maintain the version number in sync